python/cudf/cudf/core/column/column.py

```diff
@@ -1337,13 +1340,22 @@ def _label_encoding(
         """
         from cudf._lib.join import join as cpp_join
 
+        if na_sentinel is None:
```

Comment:
Is my understanding correct that if we replace this line with:

```suggestion
        if na_sentinel is None or na_sentinel.value is cudf.NA
```

Reply:
Yup, that's correct. Made the change 👍